src/components/Board/Board.js

```diff
@@ -38,12 +36,12 @@ const Board = () => {
 
     const boardCopy = [...boardArr]; // making copy so board can be updated
 
-    if (currentPlayer === playerRed) {
-      boardCopy[y][x] = "Red";
-      setCurrentPlayer(playerYellow);
-    } else if (currentPlayer === playerYellow) {
-      boardCopy[y][x] = "Yellow";
-      setCurrentPlayer(playerRed);
+    if (currentPlayer === "Red") {
```

Comment:
So... there's really no good reason to have an uppercase and a lowercase variant of the same string value. I'd strongly push you to add const variables at the top of this file that declare exactly one authoritative values:

```
const RED = 'Red';
const YELLOW = 'Yellow';
```

And with those at the top of Board.js, replace every hard-coded string `'red'` or `'Red'` with `RED`, etc.